sdcm/cluster.py

```diff
@@ -2158,6 +2158,8 @@ def offline_install_scylla(self, unified_package, nonroot):
             else:
                 install_cmds = dedent("""
                     tar xvfz ./unified_package.tar.gz
+                    echo 'export PATH="$HOME/scylladb/share/cassandra/bin:$HOME/scylladb/bin:$PATH"' >> ~/.bashrc
```

Comment:
it's mentioned in official docs that we can extend PATH with these:
https://docs.scylladb.com/manual/stable/getting-started/installation-common/unified-installer.html#configure-and-run-scylladb
also it mentions `export PATH=$PATH:~/scylladb/python3/bin` but I think we don't need it.